tests/functional/test_change_email_address.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from flask import url_for
+
+q_email = 'some.one@digital.cabinet-office.gov.uk'
+
+email_depts = [('some.one@cabinetoffice.gov.uk',
+                'Cabinet Office')]
+
+
+class When_on_change_email(object):
+
+    def it_shows_the_email_address_query_string_in_the_textbox(
+            self, live_server, browser):
+
+        browser.visit(url_for('main.change_email_address',
+                              email_address=q_email, _external=True))
+
+        assert browser.find_by_css('#email_address').value == q_email
+
+    @pytest.mark.parametrize("email_address,department", email_depts)
```

Comment:
Using `parametrize` with only one set of parameters damages readability by separating the values used in the test from the test body and making the test function signature more complex than necessary. Just have the test name/email pair as local variables in the test.